### DISCUSSION TITLE
Anybody facing following error while connecting Azure openai LLM model

### DISCUSSION THREAD

**Discussion by tiru** [MuleSoft-AI-Chain-Project/mule-ai-chain-connector#70] [General]
Anybody facing following error while connecting Azure openai LLM model. Same code working fine with openai

INFO  2024-10-20 21:15:55,524 [[MuleRuntime].uber.08: [mac-knowledge-store].mac-get-infro-from-store.CPU_LITE @4f81895d] [processor: mac-get-infro-from-store/processors/0; event: 553ae821-8f0f-11ef-bbd8-1cce511f3cba] dev.langchain4j.model.azure.AzureOpenAiChatModel: Error generating response, {error={code=404, message=Resource not found}}
ERROR 2024-10-20 21:15:55,566 [[MuleRuntime].uber.08: [mac-knowledge-store].mac-get-infro-from-store.CPU_LITE @4f81895d] [processor: mac-get-infro-from-store/processors/0; event: 553ae821-8f0f-11ef-bbd8-1cce511f3cba] org.mule.runtime.core.privileged.exception.DefaultExceptionListener: 
********************************************************************************
Message               : tokenUsage cannot be null
Element               : mac-get-infro-from-store/processors/0 @ mac-knowledge-store:mac-vector-store-demo.xml:88 (Embedding get info from store)
Element DSL           : <ms-aichain:embedding-get-info-from-store doc:name="Embedding get info from store" doc:id="913ed660-0b4a-488a-8931-26c599e859b5" config-ref="MuleSoft_AI_Chain_Config" storeName="#["C:\\Tirumurugan\\Projects\\Mule_project\\mac-demo\Stores\\" ++ payload.storeName]" getLatest="true">
<ms-aichain:data><![CDATA[
#[payload.prompt]
]]></ms-aichain:data>
</ms-aichain:embedding-get-info-from-store>
Error type            : MS-AICHAIN:EMBEDDING_OPERATIONS_FAILURE
FlowStack             : at mac-get-infro-from-store(mac-get-infro-from-store/processors/0 @ mac-knowledge-store:mac-vector-store-demo.xml:88 (Embedding get info from store))

  (set debug level logging or '-Dmule.verbose.exceptions=true' for everything)
********************************************************************************

**Comment by amirkhan-ak-sf** [comment]
Hi @tiru 

Could you please try [this branch](https://github.com/MuleSoft-AI-Chain-Project/mulesoft-ai-chain-connector/tree/add/hf).

**Reply by tiru** [reply]
Thank you for a quick response. Sure let me
